Documentation/etcd-mixin/mixin.libsonnet

```diff
@@ -219,6 +219,32 @@
               message: 'etcd instance {{ $labels.instance }} HTTP requests to {{ $labels.method }} are slow.',
             },
           },
+         {
+            alert: 'etcdDatabaseSizeAlmostFull',
```

Comment:
What if we defined this as `etcdBackendQuotaLowSpace` ? The condition for the alert is technically based on the threshold defined by quota.

Reply:
Makes sense. Fixed it, thanks.